airbyte-cdk/python/cdk-migrations.md

```diff
@@ -1,15 +1,19 @@
 # CDK Migration Guide
 
-## Importing classes
-Starting from 1.0.0, CDK classes and functions should be imported directly from `airbyte_cdk` (example: `from airbyte_cdk import HttpStream`). Lower-level `__init__` files are not considered stable, and will be modified without introducing a major release.
+## Upgrading to 3.0.0
+Version 3.0.0 of the CDK updates the `HTTPStream` class by reusing the `HTTPClient` under the hood.
 
-Introducing breaking changes to a class or function exported from the top level `__init__.py` will require a major version bump and a migration note to help developer upgrade.
+- `backoff_time` and `should_retry` methods are removed from HttpStream
+- `HttpStreamAdapterHttpStatusErrorHandler` and `HttpStreamAdapterBackoffStrategy` adapters are marked as `deprecated`
+- `raise_on_http_errors`, `max_retries`, `max_time`, `retry_factor` are marked as `deprecated`
 
-Note that the following packages are not part of the top level init because they require extras dependencies, but are still considered stable:
-- `destination.vector_db_based`
-- `source.file_based`
+### Migration of `should_retry` method
```

Comment:
Should we give an example of a migration that we've done/will be doing for another connector? "A picture is worth a thousand words"

Reply:
added